support-frontend/app/actions/CustomActionBuilders.scala

```diff
@@ -66,17 +67,27 @@ class CustomActionBuilders(
       AwsCloudWatchMetricPut(AwsCloudWatchMetricPut.client)(cloudwatchEvent)
     }
 
-    private def maybePushAlarmMetric(result: Result) =
-      if (
-        result.header.status.toString.head != '2' && !result.header.reasonPhrase.contains(
-          emailProviderRejectedCode,
-        ) && !result.header.reasonPhrase.contains(
-          invalidEmailAddressCode,
-        )
-      ) {
-        logger.error(scrub"pushing alarm metric - non 2xx response ${result.toString()}")
-        pushAlarmMetric
+    private def isNon200Result(result: Result) = result.header.status.toString.head != '2'
+
+    private def maybePushAlarmMetric(result: Result) = {
+      val ignoreList = Set(
+        emailProviderRejectedCode,
+        invalidEmailAddressCode,
+        recaptchaFailedCode,
+      )
+      if (isNon200Result(result)) {
+        if (!ignoreList.contains(result.header.reasonPhrase.getOrElse(""))) {
+          logger.error(
+            scrub"pushing alarm metric - non 2xx response. Http code: ${result.header.status}, reason: ${result.header.reasonPhrase}",
+          )
+          pushAlarmMetric
+        } else {
+          logger.info(
+            s"not pushing alarm metric for ${result.header.status} ${result.header.reasonPhrase} as it is in our ignore list",
```

Comment:
Perhaps this might be more inline with the log error above?  `not pushing ignored alarm metric - 2xx response. Http code: ${result.header.status}, reason: ${result.header.reasonPhrase}`

Reply:
This would still be a non 200 response, we're just not alarming on it because it's in the ignored list.